src/mongo_logger.ts

```diff
@@ -455,21 +461,21 @@ function attachCommandFields(
 ) {
   log.commandName = commandEvent.commandName;
   log.requestId = commandEvent.requestId;
-  log.driverConnectionId = commandEvent?.connectionId;
-  const { host, port } = HostAddress.fromString(commandEvent.address).toHostPort();
+  log.driverConnectionId = commandEvent.connectionId;
+  const { host, port } = HostAddress.fromString(commandEvent.address ?? '').toHostPort();
```

Comment:
I see the `?.` may have accidentally been left in given the input shouldn't be nullish, but why did the address change?

Reply:
How would this occur? It isn't our intention to make access to the logger public so only our "trusted" types should be able to make it into the logger.

Reply:
I see, I thought it was our intention to make the logger public to users, so that they could log anything they wished, especially since a user has access to the logger through `mongoClient.mongoLogger.`

I wrote a test case for this under `describe('defaultLogTransform', ...`

Reply:
I do not think we intend to make `client.mongoLogger` public, it doesn't make sense for something outside the driver to use the driver's logger. So can we clean up these type changes?

Reply:
I was mistaken, client.mongoLogger is already internal. I've removed the extraneous `HostAddress.fromString(commandEvent.address ?? '').toHostPort();` and the matching test.